packages/blade/src/components/Button/BaseButton/BaseButton.stories.tsx

```diff
@@ -0,0 +1,68 @@
+import type { ComponentStory, Meta } from '@storybook/react';
+import {
+  Title,
+  Subtitle,
+  Primary,
+  ArgsTable,
+  Stories,
+  PRIMARY_STORY,
+  Description,
+} from '@storybook/addon-docs';
+import iconMap from '../../Icons/iconMap';
+import type { BaseButtonIcon, BaseButtonProps } from './BaseButton';
+import BaseButtonComponent from './BaseButton';
+
+export default {
+  title: 'Components/Button/BaseButton (Internal)',
```

Comment:
💡 Possibly what we can also do is rename the internal stories with `*.stories.internal.tsx` and then in our path mapping add a dev flag which will enable these kind of stories on dev mode and will be stripped off in prod story so we can have better DX

Reply:
Done in #490